front/app/containers/Admin/flexible-pages/NavigationSettings/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Box } from '@citizenlab/cl2-component-library';
+
+// components
+import VisibleNavbarItemList from 'containers/Admin/flexible-pages/VisibleNavbarItemList';
+import HiddenNavbarItemList from 'modules/commercial/customizable_navbar/admin/containers/NavigationSettings/HiddenNavbarItemList';
+import useFeatureFlag from 'hooks/useFeatureFlag';
+
+const NavigationSettings = () => (
+  <>
+    <Box mb="44px">
+      <VisibleNavbarItemList />
+    </Box>
+
+    {useFeatureFlag({ name: 'customizable_navbar' }) && (
```

Comment:
Should I create an outlet for it?

Reply:
Yep! Then inside the module's index file (where you decide what should render in this outlet) you do the condition (the feature flag check in this case). Examples: 
* front/app/modules/commercial/similar_ideas/index.tsx
* front/app/modules/commercial/custom_maps/index.tsx